src/Eccube/Form/Type/Admin/MasterdataDataType.php

```diff
@@ -69,6 +69,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             if ($data['id'] && empty($data['name'])) {
                 $form['name']->addError(new FormError($app->trans('This value should not be blank.')));
             }
+
+            if ($data['name'] && $data['id'] == '') {
```

Comment:
Please use empty method( dont use "==" in this case.)
Do you have reason of using "=="?